sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -21,7 +21,7 @@
           <template #="{ notAllowed }">
             <v-list-item
               :to="{ name: AESTRoutesConst.REPORTS }"
-              prepend-icon="mdi-home-outline"
+              prepend-icon="fa:fa-regular fa-copy"
```

Comment:
Unfortunately, institutions and the Ministry are using different different libs. Please keep using the same lib for the file.